<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slides</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="edit_view">
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Odtwarzaj</string>
    </property>
    <addaction name="action_run"/>
    <addaction name="action_stop"/>
    <addaction name="action_prev_slide"/>
    <addaction name="action_next_slide"/>
    <addaction name="action_next_song"/>
    <addaction name="action_prev_song"/>
   </widget>
   <widget class="QMenu" name="menu_window">
    <property name="title">
     <string>Okno</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menu_window"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="playlist_dock">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>95</height>
    </size>
   </property>
   <property name="accessibleName">
    <string>Playlista</string>
   </property>
   <property name="windowTitle">
    <string>Playlista</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QListView" name="listView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dir_dock">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>93</height>
    </size>
   </property>
   <property name="accessibleName">
    <string>Widok katalogów</string>
   </property>
   <property name="windowTitle">
    <string>Widok Katalogów</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTreeView" name="directory_view">
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolbar_play">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>narzędzia</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_prev_song"/>
   <addaction name="action_prev_slide"/>
   <addaction name="action_run"/>
   <addaction name="action_stop"/>
   <addaction name="action_next_slide"/>
   <addaction name="action_next_song"/>
  </widget>
  <widget class="QDockWidget" name="preview_dock">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>150</height>
    </size>
   </property>
   <property name="accessibleName">
    <string>Podgląd</string>
   </property>
   <property name="windowTitle">
    <string>Podgląd slajdów</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QToolBar" name="toolbar_save">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_save"/>
   <addaction name="action_save_as"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>Otwórz</string>
   </property>
  </action>
  <action name="action_run">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/start.png</normaloff>:/images/images/start.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/stop.png</normaloff>:/images/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="action_next_slide">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/next.png</normaloff>:/images/images/next.png</iconset>
   </property>
   <property name="text">
    <string>Następny slajd</string>
   </property>
  </action>
  <action name="action_prev_slide">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/prev.png</normaloff>:/images/images/prev.png</iconset>
   </property>
   <property name="text">
    <string>Poprzedni slajd</string>
   </property>
  </action>
  <action name="action_next_song">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/next_song.png</normaloff>:/images/images/next_song.png</iconset>
   </property>
   <property name="text">
    <string>Następny utwór</string>
   </property>
  </action>
  <action name="action_prev_song">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/prev_song.png</normaloff>:/images/images/prev_song.png</iconset>
   </property>
   <property name="text">
    <string>Poprzedni utwór</string>
   </property>
  </action>
  <action name="action_preview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Podgląd slajdów</string>
   </property>
  </action>
  <action name="action_dir">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Widok katalogów</string>
   </property>
  </action>
  <action name="action_playlist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Playlista</string>
   </property>
  </action>
  <action name="action_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/next.png</normaloff>:/images/images/next.png</iconset>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/next_song.png</normaloff>:/images/images/next_song.png</iconset>
   </property>
   <property name="text">
    <string>Zapisz jako</string>
   </property>
  </action>
  <action name="action_new">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/start.png</normaloff>:/images/images/start.png</iconset>
   </property>
   <property name="text">
    <string>Nowy</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
